hydpy/models/hland_v1.py

```diff
@@ -41,7 +42,8 @@
 
 
 The following figure shows the general structure of H-Land Version 1.  Note that zones
-of type |FIELD| and |FOREST| are based on the same set of process equations:
+of type |FIELD| and |FOREST| are based on the same set of process equations and that
+the optional submodel |rconc_uh| is included in this scheme:
```

Comment:
I'm not sure if "optional submodel rconc_uh is included" is understandable.

Reply:
Maybe slightly changed: "In this
scheme the submodel |rconc_uh| is included (which is optional):"

Reply:
ok